### PR TITLE
[Doc](Load)Add mysql load document

### PR DESCRIPTION
**Pull request by SaintBacchus**
# Proposed changes

Issue Number: close #xxx

## Problem summary

Add mysql load document

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by hf200012** [comment]
@SaintBacchus rebase

File: docs/zh-CN/docs/data-operate/import/import-way/mysql-load-manual.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+## 基本操作举例
+
+### 创建测试表
+```sql
+CREATE TABLE testdb.t1 (pk INT, v1 INT SUM) AGGREGATE KEY (pk) DISTRIBUTED BY hash (pk) PROPERTIES ('replication_num' = '1');
+```
+
+### 导入客户端文件
+假设在客户端本地当前路径上有一个CSV文件, 名为`client_local.csv`, 使用MySQL LOAD语法讲表导入到测试表`testdb.t1`中.
```
Comment (DongLiang-0) [inline docs/zh-CN/docs/data-operate/import/import-way/mysql-load-manual.md:64]: could you change "讲表导入" to "将表导入"

Comment (SaintBacchus) [reply docs/zh-CN/docs/data-operate/import/import-way/mysql-load-manual.md:64]: OK

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+5. `PARTITION`语法支持指定分区导入
+6. `COLUMNS TERMINATED BY`指定列分隔符
+7. `LINES TERMINATED BY`指定行分隔符
+8. `IGNORE num LINES`用户跳过CSV的表头, 可以跳过任意行数. 该语法也可以用`IGNORE num ROWS`代替
+9. 列映射语法, 具体参数详见[导入的数据转换](../../../data-operate/import/import-way/mysql-load-manual.md) 的列映射章节
+10. `PROPERTIES`参数配置, 详见下文
+
+### PROPERTIES
+
+1. max_filter_ratio：最大容忍可过滤（数据不规范等原因）的数据比例。默认零容忍。
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md:70]:
```suggestion
1. `max_filter_ratio`：最大容忍可过滤（数据不规范等原因）的数据比例。默认零容忍。
```

Same for other properties

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+1. 将客户端本地文件'testData'中的数据导入到数据库'testDb'中'testTbl'的表。指定超时时间为 100 秒
+
+    ```sql
+    LOAD DATA LOCAL
+    INFILE 'testData'
+    INTO TABLE testDb.testTbl
+    PROPERTIES ("timeout"="100")
+    ```
+
+2. 将服务端本地文件'/root/testData'中的数据导入到数据库'testDb'中'testTbl'的表。指定超时时间为 100 秒
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md:91]: Rewrite this example when fixing the security issue

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+
+        ```sql
```
Comment (morningman) [inline docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md:93]: Use same indent level for all example

File: docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md (hunk 2 above)
Comment (SaintBacchus) [reply docs/zh-CN/docs/sql-manual/sql-reference/Data-Manipulation-Statements/Load/MYSQL-LOAD.md:70]: this is same with stream load docs

**Comment by SaintBacchus** [comment]
run buildall

**Comment by SaintBacchus** [comment]
run buildall

**Review by luzhijing — APPROVED** [review]
LGTM

**Pull request merged**